conf/messages-ecc.cy

```diff
@@ -69,7 +69,21 @@ cds.email-confirmed.para.get-access=TRANSLATE You can continue with your applica
 
 cds.subscription.address-details.street.label=TRANSLATE Building and street
 
-cds.subscription.outcomes.success.will-send-email=TRANSLATE We will send you an email to confirm when you have access to {0}. This can take up to two hours.
+cds.subscription.outcomes.success.will-send-email=TRANSLATE We will send you an email to confirm when you have access to [SERVICE]. This can take up to two hours.
 
 cds.registration.you-already-have-eori.title=TRANSLATE You already have an EORI
-cds.registration.you-already-have-eori.para1=TRANSLATE We have detected you already have an EORI.  You can continue and subscribe to the the {0} service.
\ No newline at end of file
+cds.registration.you-already-have-eori.para1=TRANSLATE We have detected you already have an EORI.  You can continue and subscribe to the [SERVICE] service.
+
+cds.subscription.information.title=TRANSLATE You need to subscribe
+cds.subscription.information.heading=TRANSLATE You need to subscribe to access [SERVICE]
+cds.subscription.information.subheading=TRANSLATE How to get access to [SERVICE]
```

Comment:
You'll be changing [SERVICE] to {0} when we will have translations?

This is just for content purposes to be more readable for translation team?

Reply:
Yes.  That is the idea.  I figured that the message needs to be replaced anyway so it doesn't matter if the placeholders are missing.